src/crs/geographic.jl

```diff
@@ -369,6 +441,28 @@ function Base.convert(::Type{LatLon{Datum}}, coords::Cartesian{Datum,3}) where {
   convert(LatLon{Datum}, lla)
 end
 
+function Base.convert(::Type{Cartesian{Datum}}, coords::GeocentricLatLon{Datum}) where {Datum}
+  geocentric_lla = convert(GeocentricLatLonAlt{Datum}, coords);
+  return convert(Cartesian{Datum}, geocentric_lla)
+end
+
+function Base.convert(::Type{GeocentricLatLon{Datum}}, coords::Cartesian{Datum,3}) where {Datum}
+  geocentric_lla = convert(GeocentricLatLonAlt{Datum}, coords);
+  return convert(GeocentricLatLon{Datum}, geocentric_lla)
```

Comment:
we avoid underscores in variable names, and don't use explicit return in the last statement